src/styles/main.scss

```diff
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+
```

Comment:
you may consider for future putting different scss files in different folders (styles/blocks, styles/utils), when there will be too many of them)